src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int number = random.nextInt(100);

```

Comment:
replace 100 with constant 
```suggestion
        int number = random.nextInt(100);

```

Reply:
I made constant (MAX_NUMBER) in Lottery class.
In witch class better to do this constant?
Beacose, i think that this constant discibe state of Ball(s). Maybe, it was be better made this constant in Ball class?